service-base/src/main/java/org/eclipse/hono/service/AbstractProtocolAdapterBase.java

```diff
@@ -1146,6 +1194,59 @@ protected final Message addProperties(
             final TenantObject tenant,
             final JsonObject registrationInfo,
             final Integer timeUntilDisconnect) {
+        return addProperties(msg, target, publishAddress, tenant, registrationInfo, timeUntilDisconnect, null);
+    }
+
+    /**
+     * Sets Hono specific properties on an AMQP 1.0 message.
+     * <p>
+     * The following properties are set:
+     * <ul>
+     * <li><em>to</em> will be set to the address consisting of the target's endpoint and tenant</li>
+     * <li><em>creation-time</em> will be set to the current number of milliseconds since 1970-01-01T00:00:00Z</li>
```

Comment:
please document here that this method sets the message's *ttl* header field

Reply:
I have added.